src/tests/ftest/scrubber/aggregation.py

```diff
@@ -0,0 +1,70 @@
+"""
+  (C) Copyright 2018-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from scrubber_test_base import TestWithScrubber
+from telemetry_utils import TelemetryUtils
+
+
+class TestScrubberEvictWithAggregation(TelemetryUtils, TestWithScrubber):
```

Comment:
`TelemetryUtils` is not a `Test` class and cannot be used this way.  You want to import from `TelemetryUtils` the methods you want to use.

Reply:
Yes... doing the changes to call the get_metrics method alone.

Reply:
Done.